src/utils/todoUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { Filters } from '../types/Filters';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filter: Filters): Todo[] => {
+  switch (filter) {
+    case Filters.Active:
+      return todos.filter(todo => !todo.completed);
+
+    case Filters.Completed:
+      return todos.filter(todo => todo.completed);
+
+    case Filters.All:
```

Comment:
You can remove this line and leave the default case. It does not affect anything